src/Okta.Sdk.IntegrationTests/LinkedObjectClientShould.cs

```diff
@@ -0,0 +1,323 @@
+﻿using System.Collections.Generic;
```

Comment:
Make sure to include StyleCop header. VS should hint you at the top of the file to include it.
Also, to be consistent with our other ITs let's rename this file/class to LinkedObjectScenarios. I know we have mixed names, but the rule is if the IT we are creating is following an API reference (Applications, Users, Groups, etc) the name should be <API>Scenarios, otherwise <X>Should.